aws/src/main/java/org/apache/iceberg/aws/s3/S3OutputStream.java

```diff
@@ -425,6 +425,10 @@ private void completeUploads() {
         requestBuilder.tagging(Tagging.builder().tagSet(writeTags).build());
       }
 
+      if (s3FileIOProperties.writeStorageClass() != null) {
```

Comment:
I think this should also be set in `initializeMultiPartUpload()`

Reply:
ok, got it, and we are going to test this tagging in our aws iceberg table

Reply:
more specifically, this should be set on `CreateMultipartUploadRequest.Builder` in `initializeMultiPartUpload()`